packages/nextjs/components/scaffold-eth/Contract/InheritanceTip.tsx

```diff
@@ -0,0 +1,14 @@
+export const InheritanceTip = ({ inheritedFrom }: { inheritedFrom?: string }) => {
```

Comment:
You can use implicit return here:


```ts
export const InheritanceTip = ({ inheritedFrom }: { inheritedFrom?: string }) => (
  <>
    {inheritedFrom && (
      <span
        className="tooltip tooltip-top tooltip-secondary px-2 before:content-[attr(data-tip)]"
        data-tip={`Inherited from: ${inheritedFrom}`}
      >
        *
      </span>
    )}
  </>
);

```

Reply:
Do you mean that I should move the `{inheritedFrom && (` to the components using this component? I wanted the outer components to be as clean as possible. But I can move it out.

Reply:
No, I'm just saying that you can convert your component to use "implicit return" (no need to use "{" + "return" + "("... just "("). 

Compare my code snippet to yours! You can also google "JS implicit return".

It's not a big deal, more like a code-style preference

Reply:
A, I misread your snippet to being what I had submitted. Yes indeed, I'll fix this as well. :+1:

Reply:
Fixed here: https://github.com/FilipHarald/scaffold-eth-2-sandbox/commit/7f9ef374f3e8949ce29e74635f133db74f44a7c7